packages/core-ui/package.json

```diff
@@ -47,5 +47,8 @@
     "rollup-plugin-terser": "^7.0.2",
     "rollup-plugin-typescript2": "^0.35.0",
     "rollup-plugin-uglify": "^6.0.4"
+  },
+  "dependencies": {
```

Comment:
Can this be a `peerDependency`? As we ideally don't want rollup bundling it in the package.

Reply:
Ah.. my bad! thank you